src/db/MariadbIfc.py

```diff
@@ -400,22 +400,24 @@ def getProfile(self,
         return profile
 
     def getProfiles(self,
-                    user_id : int,
-                    name : str) -> list:
+                    name    : str,
+                    rarity  : list,
+                    user_id : int) -> list:
```

Comment:
You reorganized the parameters and broke the /showprofiles getProfiles

Reply:
Fixed, and also added the ability to filer on rarity in the /showprofile command (since they use the same underlying db functions)